scripts/install-du.ps1

```diff
@@ -16,7 +17,9 @@ Param(
     [ValidateSet('Release', 'RelWithDebInfo', 'Debug')]
     [string]$Type = 'Debug',
     [ValidateSet('x64', 'arm64')]
-    [string]$Arch = 'x64',
+    [string]$HostArch = 'x64',
```

Comment:
I can understand renaming Arch to TargetArch (it's clearer) but what is "HostArch"? isn't obvious what system this script is currently running on?

Reply:
i looked below -- it would be preferable not to do any registration in this file at all, which should remove the need for a hostarch (or analyzing system)

Reply:
yes, I was trying to avoid analyzing system arch for short-term, but I think I like your registration template approach so that we don't have to do any logic other than search and replace.